src/components/dashboard/main-content/course-enrollments/course-cards/mark-complete-modal/ModalBody.jsx

```diff
@@ -13,11 +13,15 @@ const ModalBody = () => {
     <>
       {confirmError && <ModalError />}
       <p className="m-0">
-        Are you sure you want to mark
```

Comment:
I think it makes sense from a design perspective to break this text out into 2 paragraphs, e.g.:

![image](https://user-images.githubusercontent.com/2828721/84165714-0106ae80-aa42-11ea-839b-0a86d52881ee.png)